config/settings.yml

```
@@ -15,3 +15,6 @@ user:
 language:
   vietnamese: "Vietnamese"
   english: "English"
+tour:
+  per_page: 8
+  admin_per_page: 10
```

```suggestion
page:
  per_page_8: 8
  per_page_10: 10
```
chị suggest nên đổi thành như thế này để đọc tên biến là bjk giá trị luôn và còn tái sử dụng nè